cmd/archeio/main.go

```diff
@@ -17,58 +17,58 @@ limitations under the License.
 package main
 
 import (
-	"log"
+	"flag"
 	"net/http"
 	"os"
-	"regexp"
 	"strings"
+
+	"k8s.io/klog/v2"
 )
 
 func main() {
+	// cloud run expects us to listen to HTTP on $PORT
 	port := os.Getenv("PORT")
 	if port == "" {
 		port = "8080"
 	}
-	log.Printf("listening on port %s", port)
+
+	// klog setup
+	klog.InitFlags(nil)
```

Comment:
Super-nit: you may want to move this to be the very first thing you do, just in we add more "above the fold" and forget that logging isn't "ready" yet

Reply:
filed https://github.com/kubernetes-sigs/oci-proxy/pull/15 for these